### PR TITLE
Fix readme CLI reference

### PR DESCRIPTION
**Pull request by ocefpaf**
Docs update

### CODE REVIEW

**Comment by pelson** [comment]
Perfect. Thanks @ocefpaf

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ place them. If you need help getting tokens please ask on the
 [conda-forge google group](https://groups.google.com/forum/?hl=en#!forum/conda-forge).
 
 You should be able to test parts of `conda-smithy` with whatever tokens you have.
-For example, you should be able to `conda smithy github-create` without the CI service tokens.
+For example, you should be able to `conda smithy register-create` without the CI service tokens.
```
Comment (JanSchulz) [inline README.md]: This should be `register-github`

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Making a new feedstock
 <directory_of_conda_recipe>`.     For a recipe called `foo`, this creates a
 directory called `foo-feedstock`, populates it with CI setup skeletons, adds the recipe under
 `recipe` and initializes it as a git repo.
-2. **Create a github repo:** `conda smithy github-create --organization conda-forge ./foo-feedstock`.
+2. **Create a github repo:** `conda smithy register-create --organization conda-forge ./foo-feedstock`.
```
Comment (JanSchulz) [inline README.md]: `register-github`

Comment (ocefpaf) [inline README.md]: Oops!

**Pull request merged**